docs/extensibility.md

```diff
@@ -29,531 +28,630 @@ pub trait DiscoveryHandler {
-docker tag nessie:extensibility ghcr.io/<GITHUB USERNAME>/nessie:extensibility
-# Push the nessie container to your container repo
-docker push ghcr.io/<GITHUB USERNAME>/nessie:extensibility
+docker build \
+--tag=${IMAGE} \
+--file=./samples/brokers/http/Dockerfiles/standalone \
+. && \
+docker push ${IMAGE}
 ```
 
-### Create a new Configuration
-Once the nessie broker has been created (assuming `ghcr.io/<GITHUB USERNAME>/nessie:extensibility`), the next question is how to deploy it.  For this, we need to create a Configuration called `nessie.yaml` that leverages our new protocol.  
-
-Please update the yaml below to:
-* Specify a value for the imagePullSecrets. This can be any name and will correspond to a Kubernetes secret you create, which will contain your container repo credentials.  Make note of the name you choose, as this will be used later in `kubectl create secret` and `helm install` commands.
-* Specify a value for your container image that corresponds to the container repo you are using
-
+To deploy the standalone broker, we'll need to create an Akri Configuration `./samples/brokers/http/kubernetes/http.yaml` (be sure to update **image**):
 ```yaml
 apiVersion: akri.sh/v0
 kind: Configuration
 metadata:
-  name: nessie
+  name: http
 spec:
   protocol:
-    nessie:
-      nessieUrl: https://www.lochness.co.uk/livecam/img/lochness.jpg
-  capacity: 5
+    http:
+      discoveryEndpoint: http://discovery:9999/discovery
+  capacity: 1
   brokerPodSpec:
-    hostNetwork: true
-    imagePullSecrets:
-    - name: <SECRET NAME>
+    imagePullSecrets: # Container Registry secret
+      - name: SECRET
     containers:
-    - name: nessie-broker
-      image: "ghcr.io/<GITHUB USERNAME>/nessie:extensibility"
-      resources:
-        limits:
-          "{{PLACEHOLDER}}" : "1"
-  instanceServiceSpec:
-    ports:
-    - name: grpc
-      port: 80
-      targetPort: 8083
-  configurationServiceSpec:
-    ports:
-    - name: grpc
-      port: 80
-      targetPort: 8083
+      - name: http-broker
+        image: IMAGE
+        resources:
+          limits:
+            "{{PLACEHOLDER}}": "1"
+```
+
+> **NOTE** If you're using a non-public repo, you can create an `imagePullSecrets` to authenticate
+
+
+# Create some HTTP devices
+At this point, we've extended Akri to include discovery for our HTTP protocol and we've created an HTTP broker that can be deployed.  To really test our new discovery and brokers, we need to create something to discover.
+
+For this exercise, we can create an HTTP service that listens to various paths.  Each path can simulate a different device by publishing some value.  With this, we can create a single Kubernetes pod that can simulate multiple devices.  To make our scenario more realistic, we can add a discovery endpoint as well.  Further, we can create a series of Kubernetes services that create facades for the various paths, giving the illusion of multiple devices and a separate discovery service.

```

Comment:
Maybe for the next iteration, we should add a diagram here

Reply:
sounds good.  next iteration.